mdakits/twodanalysis/metadata.yaml

```
@@ -0,0 +1,82 @@
+project_name: twodanalysis
+
+
+authors:
+  - https://github.com/monjegroup/twod-analysis-kit.git/blob/main/AUTHORS.md
+
+
+maintainers:
+  - ricard1997
+
+
+
+description:
+    This MDAKit allow for the projection of lipid properties in 2D fashion. Some of these properties include 
+    lipid order parameters, membrane thickness, area per lipid, and packing defects. Furthermore, it allows to study
+    biopolymers adsorbed in the membrane interface.
+
+
+keywords:
+  - Lipids
+  - Lipids order parameters
+  - 2D projections
+  - Lipid metrics
+  - Lipid study
+  - Biopolymers
+  - Proteins
+  - Membrane
+  - RNA
+
+
+license: GPL-2.0-or-later
+
+
+project_home: https://github.com/monjegroup/twod-analysis-kit.git
+
+
+documentation_home: https://twodanalysis.readthedocs.io/en/latest/
+
+
+documentation_type: API + Quick Guide + Tutorials
+
+
+install:
+  - pip install twod-analysis-kit
+
+
+src_install:
+  - git clone https://github.com/monjegroup/twod-analysis-kit.git
+  - cd twod-analysis-kit && pip install . && cd ..
+
+import_name: twodanalysis
+
+
+python_requires: ">=3.9"
+
+
+mdanalysis_requires: ">=2.0.0"
+
+
+run_tests:
+  - pytest --pyargs twodanalysis
```

Try running in parallel on 2 cores (which we typically have in GH runners)
```suggestion
  - pytest -n 2 --durations 10 --pyargs twodanalysis
```
We also want to see the 10 slowest tests with their durations.

---

Let's start out conservative with `-n 2`. 

You can also try `-n auto`.